RELEASE-NOTES.md

```diff
@@ -9,6 +9,7 @@
 - Added `Matern12` covariance function for Gaussian processes. This is the Matern kernel with nu=1/2.
 - Progressbar reports number of divergences in real time, when available [#3547](https://github.com/pymc-devs/pymc3/pull/3547).
 - Sampling from variational approximation now allows for alternative trace backends [#3550].
+- Add capabilities to do inference on parameters in a differential equation with `DifferentialEquation`.
```

Comment:
Can you move this to the top? That will be the highlight feature of this release.